src/test/java/core/basesyntax/service/impl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ParserServiceImplTest {
+    private static ParserServiceImpl parser;
+    private List<FruitTransaction> result;
```

Comment:
let's not create a 'result' field here. You should declare it directly in your methods. Please fix the same issue in all places